rechnung/cli.py

```diff
@@ -32,6 +32,38 @@ def init():
     print("Finished.")
 
 
+@cli1.command()
+@click.argument("year", type=int)
+@click.argument("month", type=int)
+@click.option("-c", "--cid-only", help="One customer only.")
+@click.option(
+    "-d", "--dry", is_flag=True, default=False, help="Don't write the changes."
+)
+def bill_items(year, month, cid_only, dry):
+    """
+    Bill items for all active contracts (or just one with --cid-only).
+
+    The items will be added to a list, and put into an invoice, the 
+    next time create-invoices is run.
+    """
+    print(f"Billing items for month {month} in {year}.")
+    settings = get_settings_from_cwd(cwd)
+    invoice.bill_items(settings, year, month, cid_only, dry)
+
+
+@cli1.command()
+@click.argument("suffix")
+@click.option("-c", "--cid-only")
+@click.option("-f", "--force-recreate", "force", is_flag=True)
+def create_billed_invoices(suffix, cid_only=None, force=False):
+    """
+    Mass create invoices from billed items.
```

Comment:
Add params to docstring

Reply:
Doesn't make sense at this point, as the docstring is display with --help to the user. Help-Texts is the way to go for cli-functions.